sw/device/tests/crypto/cryptotest/firmware/extclk_sca_fi.h

```diff
@@ -1,4 +1,4 @@
-// Copyright lowRISC contributors.
```

Comment:
This change should be on the previous commit. I'm OK it you find it easier to squash them.

Reply:
squashed - thanks!